iCubGenova09/icub_wbd.xml

```diff
@@ -9,6 +9,7 @@
 
     <devices>
     <!-- cartesian --> 
+
```

Comment:
Can we revert this change @davidetome ?

Reply:
Since there was this conflict I kept the version on the `devel` branch

Reply:
Uhm, you should have retained the other version as `devel` doesn't have the extra empty line.
It's nitpicking of mine but I think is important to confine the mods within the `iCubTemplates` directoryl

Reply:
Thanks!